bundles/org.openhab.binding.boschshc/README.md

```diff
@@ -0,0 +1,97 @@
+# BoschSHC Binding
+
+Binding for the Bosch Smart Home Controller:
+
+## Supported Things
+
+ - Bosch In-Wall switches
```

Comment:
Can you mention the Thing Type IDs for each Thing?

Reply:
Add thing type ids for each thing